src/tool_getparam.c

```diff
@@ -686,6 +705,10 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
       }
       break;
 
+      case 'I': /* --override */
+      config->override_file_prohibit = !toggle;
```

Comment:
This should probably indent two levels like other code does after `case:`.

Reply:
Oh yes, my IDE failed at this